### PR TITLE
restore initVariable test

### PR DESCRIPTION
**Pull request by dehann**
Trying to restore failing test relating to initVariable.  Need help to restore please.

### CODE REVIEW

**Comment by dehann** [comment]
Jim, this is the one, if you can take this PR to completion please.

**Comment by jim-hill-r** [comment]
Updated the contract and fixed the test. It is now passing.

File: src/navability/services/Variable.jl [hunk 1]
```diff
@@ -115,17 +115,37 @@ function SessionKey(
...
+  #
+  Dict{String,String}(
+    "userId" => userId,
+    "robotId" => robotId,
+    "sessionId" => sessionId,
+    "variableLabel" => variableLabel
+  )
+end
+
+function VariableId(
```
Comment (dehann) [inline src/navability/services/Variable.jl:133]: what's the different 1 of 2

File: src/navability/services/Variable.jl [hunk 2]
```diff
@@ -115,17 +115,37 @@ function SessionKey(
...
+    key
+  )
+  #
+  Dict{String,Any}(
+    "key" => key
+  )
+end
+
+function VariableId(
```
Comment (dehann) [inline src/navability/services/Variable.jl:142]: whats the difference 2 of 2?

**Review by dehann — COMMENTED** [review]
hi @jim-hill-r , cool yes i follow better now what and where you are changing.  One comment, think there is a duplicate function as commented?

**Comment by dehann** [comment]
I commented out the extra function, running tests now.  Will merge if all is good.

File: src/navability/services/Variable.jl (hunk 2 above)
Comment (jim-hill-r) [reply src/navability/services/Variable.jl:142]: Ya that is dup. You removed the correct one.

Comment (dehann) [reply src/navability/services/Variable.jl:142]: Yay

image

**Pull request merged**